stdlib/_bisect.pyi

```diff
@@ -27,27 +27,21 @@ if sys.version_info >= (3, 10):
         *,
         key: Callable[[_T], SupportsRichComparisonT] = ...,
     ) -> int: ...
-    @overload
-    def insort_left(a: MutableSequence[_T], x: _T, lo: int = ..., hi: int | None = ..., *, key: None = ...) -> None: ...
-    @overload
     def insort_left(
         a: MutableSequence[_T],
```

Comment:
`_T` should probably be `SupportsRichComparisonT` here and elsewhere, but that's probably out of scope for this PR.